deploy/charts/harvester/values.yaml

```diff
@@ -322,11 +322,11 @@ jobs:
 
           ## Specify the repository of image.
           ##
-          repository: bitnami/kubectl
+          repository: rancher/harvester-upgrade
```

Comment:
maybe we can just update the job template to use the `upgrade.image.repository` value directly.

Reply:
Fixed by updating template. Thanks!